### DISCUSSION TITLE
Any ETA of the app to arrive on the PlayStore?

### DISCUSSION THREAD

**Discussion by HJMBgit84** [rafaelvcaetano/melonDS-android#449] [General]
Hi, just wondering if at this time exist an ETA to see the next app release ok the PlayStore because I an excited to see the next update 😃🙏

**Comment by rafaelvcaetano** [comment]
It will be released on the Play Store when version Beta 1.6.0 is released. However, I have no ETA for it. I could do it right now, but there are not enough new features to justify a new release. I want to add a couple of more things before that.

**Reply by HJMBgit84** [reply]
Do not worry I am anxious waiting for a new release to see and test new features but I can wait while you make Melonds even more cool 😎, thank you my friend for your hard work on this project 😊

**Comment by ajay9634** [comment]
> It will be released on the Play Store when version Beta 1.6.0 is released. However, I have no ETA for it. I could do it right now, but there are not enough new features to justify a new release. I want to add a couple of more things before that.

Please add 3d enhance option like drastic ds

And I am amazed and happy, star war jedi allience is working on it unlike drastic ds emulator.

**Comment by rafaelvcaetano** [comment]
Just leaving an update here because it's finally available on the Play Store!
https://play.google.com/store/apps/details?id=me.magnum.melonds
